### PR TITLE
[Layouts] Added separate layout for GreenCity and Ubs-admin / task #2680

### PR DESCRIPTION
**Pull request by yuriiLisunov**
related to: https://github.com/ita-social-projects/GreenCity/issues/2680

**Description:**

Made separate layout for our app and Ubs-admin page.

### CODE REVIEW

**Review by paslavskyi9roman — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

File: src/app/main/component/comments/services/comments.service.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
-import { environment } from '@environment/environment';
+import { environment } from 'src/environments/environment';
```
Comment (JimHolman1199) [inline src/app/main/component/comments/services/comments.service.ts:4]: it should be same '@environment/environment'

File: src/app/main/component/core/components/propose-cafe/propose-cafe.component.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ import { SpecificationNameDto } from '../../../../model/specification/Specificat
 import { AngularFireStorage } from '@angular/fire/storage';
 import { AngularFirestore } from '@angular/fire/firestore';
 import { Photo } from '../../../../model/photo/photo';
-import { MatSnackBarComponent } from '@global-errors/mat-snack-bar/mat-snack-bar.component';
+import { MatSnackBarComponent } from 'src/app/main/component/errors/mat-snack-bar/mat-snack-bar.component';
```
Comment (JimHolman1199) [inline src/app/main/component/core/components/propose-cafe/propose-cafe.component.ts:23]: same here

File: src/app/main/component/eco-news/components/create-edit-news/action.constants.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { ActionInterface } from '../../models/action.interface';
 import { InjectionToken } from '@angular/core';
-import { TextAreasHeight } from '@eco-news-models/create-news-interface';
+import { TextAreasHeight } from 'src/app/main/component/eco-news/models/create-news-interface';
```
Comment (JimHolman1199) [inline src/app/main/component/eco-news/components/create-edit-news/action.constants.ts:3]: same

**Comment by JimHolman1199** [comment]
please check tscofigfile to see what imports need to be corrected

**Review by Edw1x — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Comment by yuriiLisunov** [comment]
> please check tscofigfile to see what imports need to be corrected

changed

**Review by JimHolman1199 — APPROVED** [review]

**Review by SofiiaBaziv — APPROVED** [review]

**Pull request merged**